### DISCUSSION TITLE
Youtube player keeps crashing

### DISCUSSION THREAD

**Discussion by ivanjaros** [uBlockOrigin/uBlock-issues#3327] [Q&A] [answered]
For a week now, I am having trouble playing youtube videos. They usually stop playing after 30 seconds and player shows unable to load media error. This happens on live streams as well. I'd say on 25% of videos I try to play, and on some live streams(same channel) it is 100%. Probably something to do with monetization of the channel? I'd expect to see articles about new youtube changes but so far I have seen none, which concerns me and suggest it's just on my end. I though it is the new youtube change where they inject ads into the video stream directly, but I guess it is not...yet.

Anyone having issues playing videos on youtube? When I play video via FreeTube, it works just fine. I am on Firefox.

**Comment by MasterKia** [comment]
Read and follow the first post in this thread then comment there:
https://github.com/uBlockOrigin/uAssets/issues/20586

**Comment by ivanjaros** [comment]
Completely reinstalling the extension was the only thing that worked...**so far**.

**Comment by uBlock-user** [comment, marked as answer]
@stephenhawk8054 Seems he's suffering the same issue as I did

**Reply by stephenhawk8054** [reply]
@ivanjaros We found out that if your previous configuration is blocking `jnn-pa.googleapis.com`, it would cause player error: https://github.com/uBlockOrigin/uAssets/issues/20586#issuecomment-2275111776

**Reply by ivanjaros** [reply]
@stephenhawk8054 strange, i never blocked such domain. either way it works for now, after reinstall. if it happens again, i'll know where to look. might be a case where one of the public lists blocks it and after reinstalling i have not enabled it.

**Reply by ivanjaros** [reply]
@stephenhawk8054 ok, so reinstalling did not work. today i got the error again. i put the domain into whitelisted list but it did not help.

**Reply by stephenhawk8054** [reply]
Can you open new tab > reproduce the issue and on that page > click on uBO icon > 💬 [Report button](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) > scroll to the bottom > click on `Troubleshooting Information` and copy paste it here?

**Reply by Yuki2718** [reply]
And maybe check your network for e.g. DNS blocking.

**Reply by ivanjaros** [reply]
@stephenhawk8054 
```
uBlock Origin: 1.59.0
Firefox: 128
filterset (summary):
 network: 156277
 cosmetic: 48277
 scriptlet: 20746
 html: 1900
listset (total-discarded, last-updated):
 default:
  user-filters: 162-1, never
  ublock-filters: 39740-209, 3h.13m Δ
  ublock-badware: 10307-23, 3h.13m Δ
  ublock-privacy: 1040-6, 3h.13m Δ
  ublock-unbreak: 2524-16, 3h.13m Δ
  ublock-quick-fixes: 134-0, 3h.13m Δ
  easylist: 87938-49, 3h.13m Δ
  easyprivacy: 53074-61, 3h.13m Δ
  urlhaus-1: 29415-0, 17h.54m
  plowe-0: 3558-997, 17h.55m
  CZE-0: 839-13, 17h.55m
filterset (user): [array of 162 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 104 ms (selfie)
 maxAssetCacheWait: 46 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 41
 network:
  youtube.com: 21
  doubleclick.net: 3
  google.com: 15
  jnn-pa.googleapis.com: 2
 extended:
  ##.ytp-featured-product
  ###clarify-box
  ##.ytp-button.ytp-next-button
  ###sponsor-button
  ###menu > .ytd-watch-metadata.style-scope
  ###player-ads
  ##.ytp-suggested-action > button.ytp-suggested-action-badge
  ##ytd-thumbnail-overlay-toggle-button-renderer
  ##+js(trusted-replace-node-text, script, (function serverContrac…
  ##+js(adjust-setTimeout, [native code], 17000, 0.001)
  ##+js(json-prune-fetch-response, playerAds adPlacements adSlots …
  ##+js(json-prune-xhr-response, playerAds adPlacements adSlots pl…
  ##+js(set-constant, ytInitialPlayerResponse.playerAds, undefined…
  ##+js(set-constant, ytInitialPlayerResponse.adPlacements, undefi…
  ##+js(set-constant, ytInitialPlayerResponse.adSlots, undefined)
  ##+js(set-constant, playerResponse.adPlacements, undefined)
  ##+js(json-prune-fetch-response, reelWatchSequenceResponse.entri…
  ```

**Reply by stephenhawk8054** [reply]
> jnn-pa.googleapis.com: 2

> user-filters: 162-1, never

You are blocking that domain. Check your custom filters again.

**Reply by ivanjaros** [reply]
i removed it from whitelist after i posted the comment since it made no difference. that's why is in the list, most likely. anyway, i have removed all my custom filters and seems that might have been the issue. so far video si playing. i'll report if it changes. i will try to find which one it causes it, if that will be the case.

**Reply by ivanjaros** [reply]
oh, i found this in my custom filters, though i do not recall ever adding it `||jnn-pa.googleapis.com` so you might have been correct all along.

**Reply by stephenhawk8054** [reply]
>  will try to find which one it causes it, if that will be the case.

It's not about putting the domain in whitelist. You are putting that domain in "My filters" pane which would block it. Search for that in your "My filters" and remove it.

**Reply by uBlock-user** [reply]
YT no longer fires a connection request to `jnn-pa.googleapis.com`, exception can now be removed.

**Reply by Yuki2718** [reply]
I still see the request, but they now use `https://www.youtube.com/api/jnn/v1/GenerateIT` too and blocking this causes the same error. https://github.com/uBlockOrigin/uAssets/discussions/26483#discussioncomment-12716086

**Reply by uBlock-user** [reply]
I don't see the request since a month now, changes should be rolled soon to you too.

**Comment by stephenhawk8054** [comment]
@gorhill I think you can use this incident as another example of using multiple blockers as well: https://www.reddit.com/r/firefox/comments/1en0c28/people_with_yt_buffering_issues_check_your_dns_av/lh7p0vz/
